apps/remix-ide/src/app/panels/file-panel.js

```diff
@@ -96,6 +97,20 @@ module.exports = class Filepanel extends ViewPlugin {
     return this.el
   }
 
+  /**
+   *
+   * @param item { name: string, type?: string[], path?: string[], extension?: string[], pattern?: string[] }
+   * @param callback (...args) => void
+   */
+  registerContextMenuItem (item, callback) {
```

Comment:
Passing a callback might not work in the context of an iframe... (e.g I don't think that `function` is a valid parameter for a remix api
cc @bunsenstraat